provision/README.md

```diff
@@ -1,43 +1,142 @@
-# PROVISION service
+# Provision service
 
-PROVISION service provides an HTTP API to interact with Mainflux.
+Provision service provides an HTTP API to interact with Mainflux. 
+Provision service is used to setup initial applications configuration i.e. things, channels, connections and certificates that will be required for the specific use case. 
+For example lets say you are using a Mainflux and connecting gateways to it. Gateways need an easy way to configure itself for communication with Mainflux (receiving and sending controls and data) and for that you will use bootstrap service but before gateway connects to the bootstrap service configuration needs to be created (things, channels, connections, bootstrap configuration and certificates if mtls is being used). Provision service should provide an easy way of provisioning your gateways. On a gateway there can be many services running and you may require more than one thing and channel to establish all the required means of representing a gateway in your application. You may need to create any number of things and channels for your purposes, let's say that you are using services [Agent](https://github.com/mainflux/agent) and [Export](https://github.com/mainflux/export) on a gateway, if you enabled mtls each service will need a thing and certificate for access to Mainflux and you will need two channels for `Agent` (`data` and `control`) and for `Export` you could use as many as you find suitable, this kind of setup we can call `provision layout`.
```

Comment:
This part is hard to understand. I would simplify it to something like: 
`For example, to connect Gateways to Mainflux you need an easy way to setup communication channels to send and receive data or control commands. Bootstrap service can be used in this cases but you must previously create Things, Channels, Connections, Bootstrap Config and mTLS certificates. Provision service provide a way to do it so. On a Gateway...`
I'm not saying that my proposition is perfect. But your sentences are too long and with too much superfluous.